packages/backend/embedding_atlas/cli.py

```diff
@@ -130,6 +130,12 @@ def find_available_port(start_port: int, max_attempts: int = 10, host="localhost
     default=False,
     help="Allow execution of remote code when loading models from Hugging Face Hub.",
 )
+@click.option(
+    "--batch-size",
+    type=int,
+    default=32,
```

Comment:
Could the default from ST ever change? Then it might make sense to set None as the default here and pass that through.

Reply:
yeah, maybe None is safer here. cc @tomaarsen do you plan to change ST default batch size ever?

Reply:
No plans, Sentence Transformers is too mature to allow such backwards incompatible changes. I imagine it'll most likely never happen.

Reply:
Sure. That's okay if ST is stable.

Reply:
Yes, both sounds great!

Reply:
Have implemented 

- bs for images
- defaults bs for images/text (when None is passed to CLI, i.e. default behaviour)
- added more info to logging for users